tests/framework/cli_testcase.py

```diff
@@ -46,12 +51,14 @@ def clean_sharing():
         tc.submit_delete(ddata)
 
 
-def write_test_config(conf):
+def default_test_config(*args, **kwargs):
     """
-    Writes known constants and refresh tokens into config for testing
+    Returns a ConfigObj with the clitester's refresh tokens and whoami info
+    as if the clitester was logged in and a call to get_config_obj was made.
     """
     user_data = get_user_data()["clitester1a"]
-    testing_constants = {
+
+    return ConfigObj({"cli": {
```

Comment:
Can you add a comment here? The behavior of `ConfigObj`, when initialized this way, is kind of unintuitive.

Specifically, explain that:
- You can create a `ConfigObj` from a `dict`, which sets `ConfigObj.filename = None`
- When `ConfigObj.filename = None`, then `ConfigObj.write()` returns a string and writes nothing to disk

I had to go to the configobj docs site to know that this was safe.

That said, this behavior seems perfect for testing.